<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtmpConfig</class>
 <widget class="QWidget" name="AtmpConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>164</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>164</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ATMP网络设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>会  话</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>主机IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpIpEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpUserEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>密  码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ftpPswdEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="addFtpBtn">
         <property name="text">
          <string>新增</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ftpQuitBtn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
